frontend/app/Global.scala

```diff
@@ -19,7 +19,8 @@ object Global extends WithFilters(CheckCacheHeadersFilter, CacheSensitiveCSRFFil
     MasterclassEventService.start()
 
     TouchpointBackend.All.foreach(_.start())
-    MasterclassDataService.contentTask.start()
+    GuardianContentService.masterclassContentTask.start()
```

Comment:
Minor thing, might be neater to have a `GuardianContentService.start()` method that starts everything under the `GuardianContentService`